### PR TITLE
Added Top Category Section in the Home Page

### PR DESCRIPTION
**Pull request by manishbajagai2**
@smaranjitghose Finally the work's been done. Please Check it out #3 

### Screenshot : 
image

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          </div>
+          <div className="col carding m-3">
+            <img src="https://i.postimg.cc/qRXkRdP5/chaat.jpg"/>
+            <div class="info text-center mt-5">
+              <h2 className="spacingTop">Chaat and Snacks</h2>
+              <a href="#" class="btn btn-dark" role="button">Read More</a>
+            </div>
+          </div>
+        </div>
+        <div className="row">
```
Comment (smaranjitghose) [inline src/components/Categories.js:33]: Please define the headings and image paths ina separate JSON variable

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
@anushbhatia  could you assist him the vercel deployment issue due to the warnings, please?

File: src/components/Categories.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import "../styles/categories.css";
+import "../../node_modules/bootstrap/dist/css/bootstrap.min.css";
```
Comment (smaranjitghose) [inline src/components/Categories.js:3]: Please use react-bootstrap: https://react-bootstrap.github.io/getting-started/introduction/

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: src/styles/categories.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
+.title {
+  font-size: 125px;
+}
+
```
Comment (smaranjitghose) [inline src/styles/categories.css:4]: Could you please follow the structure of files requested [here](https://github.com/smaranjitghose/indokhaadyam/issues/12)

File: src/components/Categories.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+          </div>
+          <div className="col carding m-3">
+            <img src="https://i.postimg.cc/CK8h38mY/drinks.jpg"/>
+            <div class="info text-center mt-5">
+              <h2 className="spacingTop">Drinks</h2>
+              <a href="#" class="btn btn-dark" role="button">Read More</a>
+            </div>
+          </div>
+          <div className="col carding m-3">
+            <img src="https://i.postimg.cc/jSmq3c6Z/main-Course.jpg"/>
```
Comment (smaranjitghose) [inline src/components/Categories.js:49]: - All images should be placed inside `assets/categories/`
- Prefer using images from Unsplash (in webp format perhaps)

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: src/components/Categories.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+          </div>
+          <div className="col carding m-3">
+            <img src="https://i.postimg.cc/6qWB4mw5/breakfast.jpg"/>
+            <div class="info text-center mt-5">
+              <h2 className="spacingTop">Nashta <br />(Indian Breakfast)</h2>
+              <a href="#" class="btn btn-dark" role="button">Read More</a>
+            </div>
+          </div>
+          <div className="col carding m-3">
+            <img src="https://i.postimg.cc/qRXkRdP5/chaat.jpg"/>
```
Comment (anushbhatia) [inline src/components/Categories.js:26]: Do add alt text everywhere where you have added images this might solve deployment issue.

**Comment by anushbhatia** [comment]
Any updates @manishbajagai2 ?

**Pull request closed without merging**